src/Property.php

```diff
@@ -24,30 +27,25 @@ class Property extends ReflectionProperty
     /** @var bool */
     protected $isNullable = false;
 
-    /** @var bool */
-    protected $isInitialised = false;
```

Comment:
This was only set but never used and is protected. So I think that it isn't needed anymore!?

Reply:
You're right, that's an artifact from the past.